langlib/lang.boolean/src/main/ballerina/boolean.bal

```diff
@@ -24,6 +24,11 @@ import ballerina/jballerina.java;
 # and also `1` for true and `0` for `false`.
 # This is the inverse of function ``value:toString`` applied to a `boolean`.
 #
+# ```ballerina
+# string str = "true";
+# boolean booleanValue = check boolean:fromString(str);
```

Comment:
I used `check` expression instead of `boolean|error`.
What is the best way?
@mohanvive @MaryamZi

Reply:
Let's use `check`. Mentioned this in the doc too.